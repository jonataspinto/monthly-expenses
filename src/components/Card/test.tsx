import { render } from '@testing-library/react'

import { Card } from '.'

describe('<Card />', () => {
  it('should render the heading', () => {
    const { container } = render(<Card>Card</Card>)

    expect(container.firstChild).toMatchSnapshot()
  })
})
