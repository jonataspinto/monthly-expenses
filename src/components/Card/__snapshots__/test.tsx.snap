// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render the heading 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 1rem;
  border-radius: 0.5rem;
}

<div
  class="c0"
>
  Card
</div>
`
